lib/linguist/heuristics.rb

```diff
@@ -380,6 +380,24 @@ def call(data)
       end
     end
 
+    disambiguate ".pod" do |data|
+      if /^[\s&&[^\n]]*=\w+$/.match(data)
+        if /^[\s&&[^\n]]+=(begin|cut|pod)/.match(data)
+          Language["Pod 6"]
+        elsif /^=(pod|cut)/.match(data)
+          Language["Pod"]
+        elsif /^[\s&&[^\n]]*=(comment|begin pod)/.match(data)
```

Comment:
I would also add `=begin para`. Most cases a Perl 6 pod file will have =begin pod, but that is not needed if blocks certain blocks are used which don't need `=begin pod`. Many blocks can be used this way, but if it's documentation then `=begin para` is almost certain to be used to allow writing paragraphs. Unlike the case when the user does `=begin pod` in which text is processed not as Perl 6 code but as a string, if `=begin pod` is used then they well need some way to write any text that is not a `=head` title (causing them to almost certainly use `=begin para`).